mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -234,17 +234,20 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 		FIELD f_iazicyxq spawnsParticlesOnBreak Z
 		FIELD f_jnxtuzdb hardness F
 		FIELD f_jvcpotqu isAir Z
+		FIELD f_lpessbtt pistonBehavior Lnet/minecraft/unmapped/C_mojvzhua;
 		FIELD f_lsxprixg mapColor Lnet/minecraft/unmapped/C_ifcgjcqy;
 		FIELD f_mkfwgndk solidBlockPredicate Lnet/minecraft/unmapped/C_triydqro$C_izsedryc;
 		FIELD f_nblipctl material Lnet/minecraft/unmapped/C_ccpykexk;
 		FIELD f_nijrkoes emissiveLightingPredicate Lnet/minecraft/unmapped/C_triydqro$C_izsedryc;
+		FIELD f_rmobaqut offsetFunction Ljava/util/Optional;
 		FIELD f_roiqhlco toolRequired Z
 		FIELD f_vcrxfhyc shapeCache Lnet/minecraft/unmapped/C_triydqro$C_eibemhky$C_xmxhiejp;
 		FIELD f_vifiyewd blockVisionPredicate Lnet/minecraft/unmapped/C_triydqro$C_izsedryc;
 		FIELD f_vskisdzk postProcessPredicate Lnet/minecraft/unmapped/C_triydqro$C_izsedryc;
 		FIELD f_wknecqnz luminance I
 		FIELD f_wvbmkgls suffocationPredicate Lnet/minecraft/unmapped/C_triydqro$C_izsedryc;
 		FIELD f_xwxdktuf hasRandomTicks Z
+		FIELD f_yypdsdkn ignitableByLava Z
```

Comment:
this could maybe be `lavaIgnitable`, but I'll leave it up to you

Reply:
ohhh, i like that name